src/Symfony/Component/Security/composer.json

```diff
@@ -33,6 +33,7 @@
         "symfony/routing": "~2.2|~3.0.0",
         "symfony/translation": "~2.0,>=2.0.5|~3.0.0",
         "symfony/validator": "~2.5,>=2.5.5|~3.0.0",
+        "symfony/config": "~2.7|~3.0.0",
```

Comment:
I am not sure whether I should add these at all. The Security component itself neither depends or conflicts with symfony/config. Its just that for some tests the Config component is installed as a transitive dependency (probably of symfony/http-kernel) and the version is too low. 

I am very unsure about this, but I suspect that once this PR has been merged into the 2.7 branch and the raised dependency for symfony/config ~2.7 in the HttpKernel component has been picked up, we can drop this altogether?

Any advice appreciated.